internal/topology/variables/clusterclass_variable_validation.go

```diff
@@ -193,6 +149,48 @@ func validateSchema(schema *clusterv1.JSONSchemaProps, fldPath *field.Path) fiel
 	}); err != nil {
 		return append(allErrs, field.Invalid(fldPath, "", fmt.Sprintf("failed to build validator: %v", err)))
 	}
+	allErrs = append(allErrs, validateSchema(apiExtensionsSchema, fldPath)...)
```

Comment:
```suggestion

	allErrs = append(allErrs, validateSchema(apiExtensionsSchema, fldPath)...)
```
nit, just to separate it a bit from the schema validator above